lib/Jsrt/ChakraCommon.h

```diff
@@ -360,7 +360,7 @@ typedef unsigned short uint16_t;
     ///     An invalid reference.
     /// </summary>
 #ifdef __cplusplus
-    const JsRef JS_INVALID_REFERENCE = 0;
+    _Post_ _Notnull_ const JsRef JS_INVALID_REFERENCE = 0;
```

Comment:
My SAL fu is not that great- what is the intent behind this change?

Reply:
These annotations don't seem to work on non-windows builds; I believe that we have a header somewhere that `#define`s the annotations away to nothing, but it seems to be missing `_Post_`

Reply:
Sorry, this isn't needed here and I didn't realize it was checked in. Let me double-check with this removed.

Reply:
how do I find someone to help me make my first app

Reply:
i have full detailed information on my app phase 1 phase2 and phase 3 I need someone with a kind heart who wants to help